Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,7606 @@
+        """Parse for server policy dict.
+
+        Args:
+            client (Client): Fortiweb VM client.
+            policy (Dict[str, Any]): A dictionary output from API.
+
+        Returns:
+            Dict[str, Any]: Parsed dictionary.
+        """
+        parsed_data = {
+            "name": policy["_id"],
+            "deployment_mode": self.deployment_mode_api_to_user_mapper[
+                policy["depInMode"]
+            ],
+            "virtual_server": policy["virtualServer"],
+            "protocol": ",".join(
+                remove_empty_elements(
+                    [policy.get("HTTPService"), policy.get("HTTPSService")]
+                )
+            ),
+            "web_protection_profile": policy.get("InlineProtectionProfile") or "",
+            "monitor_mode": policy["MonitorMode"],
+            "http_service": policy.get("HTTPService") or "",
+            "https_service": policy.get("HTTPSService") or "",
+            "certificate": policy.get("local") or "",
+            "certificate_intermediate_group": policy.get("intergroup") or "",
+            "server_pool": policy.get("serverPool") or "",
+            "protected_hostnames": policy.get("protectedHostnames") or "",
+            "client_real_ip": policy["clientRealIP"],
+            "half_open_thresh": policy["halfopenThresh"],
+            "syn_cookie": policy["syncookie"],
+            "redirect_to_https": policy["hRedirectoHttps"],
+            "http2": policy["http2"],
+            "url_case_sensitivity": policy["URLCaseSensitivity"],
+            "comments": policy["comments"],
+        }
+        return parsed_data
+
+    def parse_custom_whitelist(
+        self, custom_whitelist: dict[str, Any]
+    ) -> dict[str, Any]:
+        """Parse for custom whitelist member dict.
+
+        Args:
+            client (Client): Fortiweb VM client.
+            custom_whitelist (Dict[str, Any]): A dictionary output from API.
+
+        Returns:
+            Dict[str, Any]: Parsed dictionary.
+        """
+        parsed_data = {
+            "id": custom_whitelist["_id"],
+            "type": self.custom_whitelist_api_to_user_mapper[custom_whitelist["type"]],
+            "name": custom_whitelist.get("itemName") or "",
+            "status": custom_whitelist["enable"],
+            "request_type": dict_safe_get(
+                self.request_type_api_to_user_mapper,
+                [custom_whitelist.get("requestType")],
+            )
+            or "",
+            "request_url": custom_whitelist.get("requestURL") or "",
+            "domain": custom_whitelist.get("domain") or "",
+            "path": custom_whitelist.get("path") or "",
+        }
+        return parsed_data
+
+    @property
+    def sub_object_id_key(self) -> str:
+        """Hold sub object id key in Fortiweb V1
+
+        Returns:
+            str: Sub object id key
+        """
+        return "_id"
+
+    @property
+    def action_user_to_api_mapper(self) -> dict[str, Any]:
```

Comment:
```suggestion
    def action_user_to_api_mapper(self) -> dict[str, int]:
```

here and everywhere else, following our discussion